tests/api_fastapi/core_api/routes/ui/test_structure.py

```diff
@@ -450,6 +279,196 @@ def test_should_return_200(self, test_client, params, expected):
+                    "label": "task_2",
+                    "tooltip": None,
+                    "setup_teardown_type": None,
+                    "type": "task",
+                    "operator": "EmptyOperator",
+                    "asset_condition_type": None,
+                },
+                {
+                    "children": None,
+                    "id": f"asset:{asset3_id}",
+                    "is_mapped": None,
+                    "label": "s3://dataset-bucket/example.csv",
+                    "tooltip": None,
+                    "setup_teardown_type": None,
+                    "type": "asset",
+                    "operator": None,
+                    "asset_condition_type": None,
+                },
+                {
+                    "children": None,
+                    "id": "sensor:dag_with_multiple_versions:dag_with_multiple_versions:external_task_sensor",
+                    "is_mapped": None,
+                    "label": "external_task_sensor",
+                    "tooltip": None,
+                    "setup_teardown_type": None,
+                    "type": "sensor",
+                    "operator": None,
+                    "asset_condition_type": None,
+                },
+                {
+                    "children": None,
+                    "id": "trigger:external_trigger:dag_with_multiple_versions:trigger_dag_run_operator",
+                    "is_mapped": None,
+                    "label": "trigger_dag_run_operator",
+                    "tooltip": None,
+                    "setup_teardown_type": None,
+                    "type": "trigger",
+                    "operator": None,
+                    "asset_condition_type": None,
+                },
+                {
+                    "children": None,
+                    "id": "and-gate-0",
+                    "is_mapped": None,
+                    "label": "and-gate-0",
+                    "tooltip": None,
+                    "setup_teardown_type": None,
+                    "type": "asset-condition",
+                    "operator": None,
+                    "asset_condition_type": "and-gate",
+                },
+                {
+                    "children": None,
+                    "id": "asset1",
+                    "is_mapped": None,
+                    "label": "asset1",
+                    "tooltip": None,
+                    "setup_teardown_type": None,
+                    "type": "asset",
+                    "operator": None,
+                    "asset_condition_type": None,
+                },
+                {
+                    "children": None,
+                    "id": "asset2",
+                    "is_mapped": None,
+                    "label": "asset2",
+                    "tooltip": None,
+                    "setup_teardown_type": None,
+                    "type": "asset",
+                    "operator": None,
+                    "asset_condition_type": None,
```

Comment:
Why do asset 1 and asset 2 have not the same structure as the asset 3 ? (we are missing the asset id in the `"id"` etc.)?

Reply:
thanks, this is wrong. rewrite the last part of structure